python_modules/libraries/dagster-aws/dagster_aws/redshift/resources.py

```diff
@@ -47,7 +33,14 @@ def execute_queries(
         pass
 
 
-class RedshiftResource(_BaseRedshiftResource):
+class RedshiftClient(BaseRedshiftClient):
+    def __init__(self, conn_args: Dict[str, Any], autocommit: Optional[bool], log: Logger):
```

Comment:
this change is breaking because the init signature changes - if we think that users are constructing `RedshiftClient` themselves I can hack around this to keep the `context`-requiring signature, but it's pretty painful to build a context when we don't have one

Reply:
we don't export it. I think it is fine.